src/Event/ValueObjects/Status.php

```diff
@@ -0,0 +1,80 @@
+<?php declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Event\ValueObjects;
+
+use Broadway\Serializer\SerializableInterface;
+use CultuurNet\UDB3\Model\ValueObject\Translation\Language;
+use InvalidArgumentException;
+
+final class Status implements SerializableInterface
+{
+    /**
+     * @var StatusType
+     */
+    private $statusType;
+
+    /**
+     * @var StatusReason[]
+     */
+    private $statusReason;
+
+    public function __construct(StatusType $statusType, array $statusReason)
+    {
+        $this->ensureTranslationsAreUnique($statusReason);
+        $this->statusType = $statusType;
+        $this->statusReason = $statusReason;
+    }
+
+    public function getStatusType(): StatusType
```

Comment:
Nit: `getStatusType()` and `getStatusReason()` could also be shortened to `getType()` and `getReason()` since this class is already called `Status`